x-pack/plugins/infra/public/components/asset_details/components/top_processes_tooltip.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiText, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+const DOCUMENTATION_LINK =
+  'https://www.elastic.co/guide/en/observability/current/view-infrastructure-metrics.html';
+const SYSTEM_INTEGRATION_DOCS_LINK = 'https://docs.elastic.co/en/integrations/system';
+
+export const TopProcessesTooltipContent = React.memo(() => {
```

Comment:
Might be better to name the file and component inside using the same name, but I'm not sure how strictly we follow this convention usually

Reply:
After moving the popup here I renamed it to `TopProcessesTooltip` which matches the file name